engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/bv/future/FutureValidatorForReadableInstant.java

```diff
@@ -1,41 +0,0 @@
-/*
- * Hibernate Validator, declare and validate application constraints
- *
- * License: Apache License, Version 2.0
- * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
- */
-package org.hibernate.validator.internal.constraintvalidators.bv.future;
-
-import javax.validation.ConstraintValidator;
-import javax.validation.ConstraintValidatorContext;
-import javax.validation.constraints.Future;
-
-import org.hibernate.validator.constraintvalidation.HibernateConstraintValidatorContext;
-import org.hibernate.validator.spi.time.TimeProvider;
-import org.joda.time.ReadableInstant;
-
-/**
- * Check if Joda Time type who implements {@code import org.joda.time.ReadableInstant} is in the future.
```

Comment:
No, we are still supporting it. I moved the class and changed the implementation so git wasn't able to detect it's a move.